blob/service.go

```diff
@@ -198,10 +198,10 @@ func (s *Service) getByCommitment(
 	}
 
 	var (
-		rawShares = make([]share.Share, 0)
+		rawShares = make([]shares.Share, 0)
 		proofs    = make(Proof, 0)
-		amount    int
-		blobShare *shares.Share
+		// expanded specifies whether blob is expanded into multiple rows
+		expanded bool
```

Comment:
let's call it something else like `spansMultipleRows bool`

Reply:
or `overflowsRow`

Reply:
renamed